templates/web/panels/panel.tt.html

```diff
@@ -149,6 +149,17 @@ <h4 class="panel_title">
                     <img src="[% image_element.url %]" alt="[% image_element.alt %]">
                 [% IF image_element.link_url.defined %]</a>[% END %]
 
+                [% IF image_element.source_url.defined %]
+                <div class="image_source" style="font-style:italic">[% lang("source") %][% sep %]:
+                    <a href="[% text_element.source_url %]">[% image_element.source_text %]
```

Comment:
isn't it `href="[% image_element.source_url %]"` ? or is it the IF above ill defined ? I'm a bit confused…

Reply:
ah right, my mistake, thanks!